Simplicity-TR.tm

```diff
@@ -9078,29 +9078,52 @@
   <math|<text|<samp|right-extend>><rsub|1,2<rsup|m>>\<assign\><text|<samp|left-extend>><rsub|1,2<rsup|m>>
   :<2><rsup|>\<vdash\><2><rsup|2<rsup|m>>> for <math|0\<leq\>n\<less\>m>
 
+  <subsubsection|<samp|left-shift-with>>
+
+  Left shift by a given amount. Bits are filled with the provided value.
+
+  <\math>
+    <rep|<text|<samp|'left-shift-with'>><rsub|2<rsup|n>>|>\<assign\><verbatim|<around*|[|110|]>><rsub|<2>>\<cdummy\><rep|<value|subsection-nr>|>\<cdummy\><rep|<value|subsubsection-nr>|>\<cdummy\><rep|n+1|>
+  </math>
+
+  <math|<text|<samp|left-shift-with>><rsub|2<rsup|n>>
+  :<2>\<times\><around*|(|<2><rsup|2<rsup|<around*|\<lceil\>|lg<around*|(|n|)>|\<rceil\>>>>\<times\><2><rsup|2<rsup|n>>|)>\<vdash\><2><rsup|2<rsup|n>>>
```

Comment:
1f245eccc7593859bcdcbfe24d0c09f4c8443867: `2 × (2^{2^ceil(lg n)}) × 2^{2^n} → 2^{2^n}`
Does `lg n` mean `log10 n`? That is surprising to me. I expected something like `2 × (2^n × 2^{2^n} → 2^{2^n}`

Reply:
`lg` is the standard CS notation for base 2 logarithms.

Reply:
Apparently the ISO wants us to use `lb` which is a notation I have never seen before: https://en.wikipedia.org/wiki/Binary_logarithm#Notation